### ISSUE TITLE
Unable to access external or internal ports

### ISSUE THREAD

**Issue by shaobinyao** [Shardeum/shardeum-bug-reporting#477]
##### What is the issue?
Unable to access external or internal ports. Please provide proper port access
  to the external and internal ports of shardeum  (9001 / 10001 are the
  defaults)

:red_circle:  Describe the error in simple terms. :red_circle: 
I only upgraded version to 1.9.1,server is not change anything,and version 1.8.9 don't have this problem.
    
#### What impact does the issue have?

:red_circle: Describe if the bug is a visual warning or if it breaks functionality causing a system to fail. :red_circle: 

##### How to reproduce the issue? 

:red_circle: 
Explain in as much detail as possible to help reproduce the bug:
```
    -where the error is seen
    -how the error was potentially caused if possible (mention previous steps taken to see the error if relevant)
    -share code and explain how the code was run
```
:red_circle: 

##### What other resources can you share regarding this issue?

:red_circle: Photos and links related to the error. :red_circle:

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.

**Comment by shaobinyao** [comment]
still error













小马
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shardeum/shardeum-bug-reporting"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月4日(星期天) 上午7:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shardeum/shardeum-bug-reporting] Unable to access external or internal ports (Issue #477)





 
update to the latest node its solved in latest version.
